spec/services/push_notifications/send_spec.rb

```diff
@@ -91,4 +88,35 @@
       sidekiq_assert_enqueued_jobs(1, only: PushNotifications::DeliverWorker)
     end
   end
+
+  context "when associated to operational vs non-operational ConsumerApp" do
+    it "doesn't create any new (RPush) Push Notifications if non-operational" do
+      user3 = create(:user)
```

Comment:
I find `user3` mildly confusing since it's the only user in this spec but I understand if you don't want to shadow the global one (I usually do in cases like this).

Reply:
Looking back to this I would say I was trying to be "overly cautious", but you're right in thinking there's no need for `user3` here.

"Overly cautious" in the sense that I really wanted this user to be "clean" (no devices or any other notifications), but since each spec runs independently within transactions the `user` would be assured to be "clean". I was just a bit too paranoid here 😅 

I'm pushing this change here soon to simplify the spec. It's also likely this will be more performant (very very slightly) without `user3` because it means fewer records/models created.